javascript-light-bulb/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Javascript Light Bulb</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body class="body-class">
+    <main>
+      <div class="wrapper">
+        <button onclick="handleOffOrOn()" class="circle"></button>
```

Comment:
You shouldn't be using `onclick`. We should be adding event listeners in JavaScript with `addEventListener`